build.sbt

```diff
@@ -1,26 +1,25 @@
-name := "health-check-adapter"
-
-version := "1.0"
+lazy val healthCheckAdapter = project
+  .copy(id = "health-check-adapter")
+  .in(file("."))
+  .enablePlugins(AutomateHeaderPlugin, GitVersioning)
 
-scalaVersion := "2.11.7"
-
-val jacksonVersion = "2.6.3"
-libraryDependencies += "com.fasterxml.jackson.core" % "jackson-core" % jacksonVersion
-libraryDependencies += "com.fasterxml.jackson.core" % "jackson-databind" % jacksonVersion
-libraryDependencies += "com.fasterxml.jackson.dataformat" % "jackson-dataformat-yaml" % jacksonVersion
-libraryDependencies += "com.fasterxml.jackson.module" %% "jackson-module-scala" % jacksonVersion
+name := "health-check-adapter"
 
-val akkaCoreVersion = "2.4.2"
-libraryDependencies += "com.typesafe.akka" %% "akka-actor" % akkaCoreVersion
-libraryDependencies += "com.typesafe.akka" %% "akka-contrib" % akkaCoreVersion
-libraryDependencies += "com.typesafe.akka" %% "akka-testkit" % akkaCoreVersion % "test"
+libraryDependencies ++= Vector(
```

Comment:
I think this is in order to make manage multi module projects easier. You would define all your dependencies in `Dependencies.scala` but not all your submodules would use all dependencies.